tests/app/framework/survey_runner_test_case.py

```diff
@@ -25,6 +26,6 @@ def setUp(self):
             with c.session_transaction() as sess:
                 sess['user_id'] = '1'
 
-        schema_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, "0_star_wars.json"))
+        schema_file = open(os.path.join(_get_schema_path, "0_star_wars.json"))
```

Comment:
Use _get_schema_file_path here, anywhere else this is happening you could do the same replacement?